docs/concepts/gateway/index.md

```diff
@@ -374,6 +371,30 @@ curl http://localhost:12345/status
 }
 ```
 
+(gateway-logging-configuration)=
+## Custom logging configuration
+
+The {ref}`Custom logging configuration <logging-configuration>` section describes customizing the logging configuration for all entities of the `Flow`.
+The `Gateway` logging can also be individually configured using a custom `logging.json.yml` file as in the below example. The custom logging file
+`logging.json.yml` is described in more detail in the {ref}`Custom logging configuration <logging-configuration>` section.
+
+````{tab} Python
+```python
+from jina import Flow
+
+f = Flow().config_gateway(log_config='./logging.json.yml')
+```
+````
+
+````{tab} YAML
+```yaml
+jtype: Flow
+gateway:
+  log_config: './logging.json.yml'
```

Comment:
is it always a file or can it be a dictionary?

Reply:
The type is a string which can refer to the jina configuration substring which are present in the `resources` folder like default, json, file, profile, quiet, remove. If the string doesn't match then the current logic is to open a file matching the python logging config schema.

Reply:
I intentionally didn't refer to the existing logging configs in the `resources` folder because I don't know how general the configs are.

Reply:
would it make sense to support directly a `Dictionary` that can update some default configs?

Reply:
We could. I'm not sure if the updates are transparent enough to the user. It might be better for the k8s deployment configuration.